### PR TITLE
Refactor building ufs block options

### PR DESCRIPTION
**Pull request by beinan**
### What changes are proposed in this pull request?
Refactor building ufs block options

### Why are the changes needed?

Make the code more clean and concise

### Does this PR introduce any user facing changes?

N/A

### CODE REVIEW

**Review by dbw9580 — APPROVED** [review]
LGTM

File: core/client/fs/src/main/java/alluxio/client/file/options/InStreamOptions.java [hunk 1]
```diff
@@ -142,30 +143,31 @@ public BlockInfo getBlockInfo(long blockId) {
    * @return a {@link Protocol.OpenUfsBlockOptions} based on the block id and options
    */
   public Protocol.OpenUfsBlockOptions getOpenUfsBlockOptions(long blockId) {
-    Preconditions.checkArgument(mStatus.getBlockIds().contains(blockId), "blockId");
-    boolean readFromUfs = mStatus.isPersisted();
+    Preconditions.checkArgument(mStatus.getBlockIds().contains(blockId),
```
Comment (jiacheliu3) [inline core/client/fs/src/main/java/alluxio/client/file/options/InStreamOptions.java:146]: Interestingly this `getBlockIds()` returns a List, but on 2nd thought just `O(n)` scan a list is probably faster than building a set.

File: core/client/fs/src/main/java/alluxio/client/file/options/InStreamOptions.java [hunk 2]
```diff
@@ -142,30 +143,31 @@ public BlockInfo getBlockInfo(long blockId) {
...
+        "block id does not belong to the file");
```
Comment (jiacheliu3) [inline core/client/fs/src/main/java/alluxio/client/file/options/InStreamOptions.java:147]:
```suggestion
        "block id %s does not belong to the file %s", blockId, mStatus.getPath());
```

**Review by jiacheliu3 — APPROVED** [review]
LGTM with a minor comment, thanks!

File: core/client/fs/src/main/java/alluxio/client/file/options/InStreamOptions.java (hunk 1 above)
Comment (beinan) [reply core/client/fs/src/main/java/alluxio/client/file/options/InStreamOptions.java:146]: Yep, the blockId list should be small for most of files

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Pull request merged**